package.json

```diff
@@ -17,11 +17,17 @@
   "homepage": "https://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
+    "test:node-require": "node test/require/node.js",
+    "build:browserify-test": "browserify test/require/browserify.js -o build/temp/browserify.js",
+    "build:webpack-test": "webpack --hide-modules test/require/webpack.js build/temp/webpack.js",
     "copy-fonts": "shx cp -R node_modules/videojs-font/fonts build/temp/font",
     "copy-dist": "shx cp -R build/temp/* dist/",
-    "copy-a11y": "shx cp sandbox/descriptions.html.example sandbox/descriptions.html",
     "copy-examples": "shx cp -R docs/examples dist/",
-    "copy-lang": "shx cp -R lang/* dist/lang/",
+    "build:lang": "npm-run-all build:lang:*",
+    "build:lang:js": "vjslang --dir dist/lang",
+    "build:lang:copy": "shx --verbose cp -R lang/* dist/lang/",
```

Comment:
why `--verbose`?

Reply:
I ran into issues with it previously and forgot to remove it, I think we can remove it now.